sample/chat-demo-mpp/desktop/build.gradle.kts

```diff
@@ -1,25 +1,25 @@
-import org.jetbrains.compose.compose
 import org.jetbrains.compose.desktop.application.dsl.TargetFormat
 
 plugins {
-    kotlin("multiplatform")
-    id("org.jetbrains.compose")
-    kotlin("plugin.serialization")
+    id(libs.plugins.kotlin.multiplatform.get().pluginId)
```

Comment:
This can be the same way as other plugins `alias(libs.plugins.kotlin.multiplatform)`

Reply:
Right thanks, forgot that

Reply:
Actually, I remembered I can't use it that way, because buildSrc is making some problems if you use the alias and include Kotlin as a dependency

Reply:
I see, maybe buildSrc can be changed in the future, it's not a best practice anymore haha

https://github.com/Kotlin/multiplatform-library-template/tree/main/convention-plugins